src/pages/UserEnrollments/UserEnrollments.css

```diff
@@ -0,0 +1,42 @@
+.user-enrollments-page {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+.header h2 {
+  margin-bottom: 0;
+}
+
+.enrollments-section {
+  flex-grow: 1;
+  padding: 1em 0;
+}
+
+.loading {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+}
+
+.organisation {
+  font-weight: 600;
+  font-size: large;
+  margin: 1em 0;
+}
+
+.enrollments-list {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1em;
+}
```

Comment:
didn't find why but there is a strange white space in the detail view:

![image](https://user-images.githubusercontent.com/4190728/68677992-5a1a6c80-055d-11ea-9e40-0854fbbd8330.png)